### DISCUSSION TITLE
如何更新到最新版本，需要删除老版本后重新下载安装吗

### DISCUSSION THREAD

**Discussion by chaoduuuu** [lencx/ChatGPT#562] [Q&A]
如何更新到最新版本，需要删除老版本后重新下载安装吗？
通过“ChatGPT”——"Chack For Update"，查看更新后，点击过同意，但是并没有更新。
电脑为mac book air

**Comment by lencx** [comment]
正常情况是可以自动更新，如果更新没反应可能是网络问题，你可以选择手动下载。之前的应用直接会被替换掉，不需要删除。

**Comment by chaoduuuu** [comment]
谢谢答复！
